paimon-core/src/main/java/org/apache/paimon/table/source/DataSplit.java

```diff
@@ -75,6 +75,16 @@ public boolean isStreaming() {
         return isStreaming;
     }
 
+    public long getLatestFileCreationTime() {
```

Comment:
Just return `Optional<Long>`. Let caller decide how to deal with `Optional.empty()`.